modules/user_accounts/test/TestPlan.md

```diff
@@ -81,9 +81,5 @@ When creating or editing a user: (subtest: edit_user)
 
 ### Widget registration on the dashboard page
 
-36. Verify that if a user has 'User Management / Survey Participant Management' permission, the number of pending
-    account approvals is displayed in the My Task panel of the dashboard page. This should be the number of entries
-    in the User Account page with the following Selection Filter: Site set to the user's site and Pending Approval
-    set to 'Yes'. The Site displayed will be 'All user sites'. Check that you are taken to that page (with the
-    Selection Filter correctly set) when you click on the task.
+36. Verify that if a user has 'Across all sites create and edit users' permission, the number of pending account approvals is displayed in the My Task panel of the dashboard page. This should be the number of entries in the User Account page with the following Selection Filter: Site set to any site except Data Coordinating Center and Pending Approval set to 'Yes'. The Site displayed will be 'All user sites'. Check that you are taken to that page (with the Selection Filter correctly set) when you click on the task.
```

Comment:
These are 2 different tests. If you have across all sites, there should be no site taken into account, and if you have user accounts without across all sites, it should only be your sites.

Reply:
Is user_accounts_multisite sufficient to access the user_accounts page? Currently, a user with `user_accounts_multisite` and without `user-accounts` permission gets a `You do not have access to this page` on User Accounts page.